### PR TITLE
Use gradient instead of image online

### PR DESCRIPTION
**Pull request by mbrandonw**
A quick (kinda hacky) fix...

### CODE REVIEW

File: Sources/PointFree/Home.swift [hunk 1]
```diff
@@ -44,10 +44,10 @@ private let episodeRowView = View<Episode> { ep in
       gridColumn(sizes: [.mobile: 12, .desktop: 7], episodeInfoColumnView.view(ep)),
 
       gridColumn(sizes: [.mobile: 12, .desktop: 5], [`class`([Class.grid.first(.mobile), Class.grid.last(.desktop)])], [
-        div([`class`([Class.size.height100pct]), style(lineHeight(0))], [
+        div([`class`([Class.size.height100pct]), style(lineHeight(0) <> gradient <> minHeight(.px(300)))], [
           a([href(path(to: .episode(.left(ep.slug))))], [
             img(
-              src: "https://waltpaper.com/wp-content/uploads/2017/06/6c7b04501f6e98ab94982a2743ede747-1024x768.jpeg",
```
Comment (mbrandonw) [inline Sources/PointFree/Home.swift]: don't use this online image

File: Sources/PointFree/Home.swift [hunk 2]
```diff
@@ -44,10 +44,10 @@ private let episodeRowView = View<Episode> { ep in
       gridColumn(sizes: [.mobile: 12, .desktop: 7], episodeInfoColumnView.view(ep)),
 
       gridColumn(sizes: [.mobile: 12, .desktop: 5], [`class`([Class.grid.first(.mobile), Class.grid.last(.desktop)])], [
-        div([`class`([Class.size.height100pct]), style(lineHeight(0))], [
+        div([`class`([Class.size.height100pct]), style(lineHeight(0) <> gradient <> minHeight(.px(300)))], [
```
Comment (mbrandonw) [inline Sources/PointFree/Home.swift]: have to do the `min-height` cause without an image the div collapses in mobile with no image.

Comment (stephencelis) [reply Sources/PointFree/Home.swift]: Wonder if we can set an aspect ratio to prevent glitches in general.

Comment (mbrandonw) [reply Sources/PointFree/Home.swift]: I tried a few approaches towards that, none of em worked great

**Pull request merged**